### PR TITLE
Hawkular-metrics doc update

### PR DESCRIPTION
**Pull request by jotak**
- Fix some broken links
- Remove old grafana integration page
- Fix some typo
- Update about some "upcoming" features now released
- Add information on String type (with the help of @jsanda )

### CODE REVIEW

File: src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ Hawkular Metrics supports common metric types including:
 
 *Tagging* +
 Hawkular Metrics provides flexible tagging support that makes it easy to organize and group data. Tagging can also be
-used provide additional information and context about data. For example, each metric
+used provide additional information and context about data.
```
Comment (pilhuhn) [inline src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc]: used to provide

File: src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc [hunk 2]
```diff
@@ -193,13 +191,16 @@ compute and maintain the total count.
...
 
-NOTE: It is currently not possible to get rate data points for gauges. This will change in an upcoming release so that
-gauges can then be used for incrementing counters. See
-ink:https://issues.jboss.org/browse/HWKMETRICS-365[HWKMETRICS-365] for details.
+==== String
+The String metric type stores any arbitrary strings. Hawkular Metrics already has an Availability metric type, but it is
+limited to a predefined number of values which cannot easily be changed. In some cases a String type with arbritrary values
+would better fit for availability events. It can also be used for storing and tracking other types of events.
+
+TIP: Note that there is a 2 KB limit for string data points.
```
Comment (pilhuhn) [inline src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc]: This should perhaps be something stronger like NOTE or IMPORTANT

**Comment by jotak** [comment]
@pilhuhn done - I flagged the note as important since it is a limitation

**Comment by pilhuhn** [comment]
Thanks @jotak

**Pull request merged**